configs:
  mode:
    runner: "rs"
    temporalSubtraction: false # Only for single vision sensor ("rs", "offimg", "offvid")
  sensor:
    general:
      fpsBoost: true
      brightness:
        alpha: 1.0
        beta: 0.0
    realSense:
      fps: 30
      resolution:
        width: 640
        height: 480
  algorithm:
    process:
      channel: "g" # ["b", "g", "r", "all"]
      subtractRL: true
      colorRange:
        hsv_green:
          lower: [35, 120, 50] # [35, 120, 50]
          upper: [85, 255, 255] # [85, 255, 255]
    postprocess:
      threshold:
        size: 90
        method: "binary" # ["binary", "otsu", "both"]
      invertBinary: true
      erosionKernel: 1
      gaussianKernel: 1
  marker:
    structure:
      size: 0.1
    detection:
      dictionary: "DICT_ARUCO_ORIGINAL"
